<launch>
    <!-- Launch vertical_roi node -->
    <node name="vertical_detector_node" pkg="cavity_detection" type="vertical_roi" output="screen"/>

    <!-- Launch horizontal_roi node -->
    <node name="horizontal_detector_node" pkg="cavity_detection" type="horizontal_roi.py" output="screen"/>
</launch>